db/migrate/20230704091131_update_id_users.rb

```diff
@@ -0,0 +1,16 @@
+class UpdateIdUsers < ActiveRecord::Migration[7.0]
```

Comment:
Should we change the name to:
```suggestion
class UpdateIdColumnOfUsers < ActiveRecord::Migration[7.0]
```
This is recommended by [Database section on Compass](https://www.rubyguides.com/2019/11/rails-flash-messages/#:~:text=A%20flash%20message%20is%20a,User%20not%20found%E2%80%9D%20(error))